### ISSUE TITLE
scores drop to zero

### ISSUE THREAD

**Issue by yossitsadok** [bourdakos1/Custom-Object-Detection#9]
Trying to reproduce your demo, I run your training example with your images and "faster_rcnn_resnet101.config" . Although the loss drops gradually the prediction scores drop fast to zero after few hundred of steps. Changing the learning rate didn't solved it.

Do you have any suggestion?

**Comment by bourdakos1** [comment]
How many images are you using to train and how many classes of objects? Also, how does your dataset compare to the coco dataset?

**Comment by yossitsadok** [comment]
I was using the dataset you provided (Star Wars images) and faster_rcnn_resnet101_coco as the base model. I'm using the default parameters and definitions that in your config file faster_rcnn_resnet101.config

**Comment by bourdakos1** [comment]
Oh, okay, maybe try running it longer? It should work to train the provided dataset out of the box, I recommend at least running it for 1,000 steps to see if it goes up at all. If you have any screenshots of the output that might help me help you a bit more
